cassandra-diagnostics-core/src/main/java/io/smartcat/cassandra/diagnostics/module/slowquery/SlowQueryConfiguration.java

```diff
@@ -1,76 +1,85 @@
 package io.smartcat.cassandra.diagnostics.module.slowquery;
 
 import java.util.ArrayList;
-import java.util.Arrays;
+import java.util.Collections;
 import java.util.List;
+import java.util.Map;
 
-import org.apache.commons.lang3.StringUtils;
+import org.yaml.snakeyaml.Yaml;
 
-import io.smartcat.cassandra.diagnostics.module.ModuleConfiguration;
+import io.smartcat.cassandra.diagnostics.config.ConfigurationException;
 
 /**
  * Typed configuration class with reasonable defaults for this module.
  *
  */
 public class SlowQueryConfiguration {
 
-    private static final String DEFAULT_SLOW_QUERY_THRESHOLD = "25";
-    private static final String DEFAULT_LOG_ALL_QUERIES = "true";
-
-    private final int slowQueryThreshold;
+    /**
+     * A helper class for constructing immutable outer class.
+     */
+    public static class Values {
+        private static final int DEFAULT_SLOW_QUERY_THRESHOLD = 25;
+        private static final boolean DEFAULT_LOG_ALL_QUERIES = true;
```

Comment:
Note to self and others, think this log all queries we added is stupid since threshold 0 should do same thing